### DISCUSSION TITLE
Mapping with inconsistent primary element name

### DISCUSSION THREAD

**Discussion by ivanmoralesv** [engram-design/FeedMe#870] [Ideas]
I need to import the latest articles added to Pocket,
but unfortunately the Pocket API JSON passes items with ID.

There is a way to get all the childrens of "list" as primary element for mapping?

Something like: list/*
Thanks!

![pocket_json](https://cloud.githubusercontent.com/assets/1270228/9327107/c22b0b64-459e-11e5-83bd-6ee4cadc0154.jpg)

**Comment by engram-design** [comment]
@ivanmoralesv Interesting, I think I can work something like this into the next release for you. Are you after this urgently?

**Comment by ivanmoralesv** [comment]
Not too much :)
Thanks for your job!

**Comment by engram-design** [comment]
No worries - will keep you informed!
